activestorage/app/models/active_storage/variant.rb

```diff
@@ -91,6 +91,14 @@ def filename
     ActiveStorage::Filename.new "#{blob.filename.base}.#{variation.format.downcase}"
   end
 
+  def content_type_for_serving
```

Comment:
juste like with `def filename` above i've implemented the missing methods, since we do not deal with a real blob.

My theory is that `tracking variants` became the default at some point in rails, and the proxycontroller evolved, without taking into account the untracked variants use case

( Indeed, tests were missing that would have caught this)

Should I open a separate for this, or since it's all related to ProxyController and `representations` as a whole, I can keep my second commit on this PR?

Reply:
Non-tracked variants not working with `ActiveStorage::Representations::ProxyController` is definitely an issue by itself.  So let's fix that in a separate PR and merge that PR first.

Reply:
done, extracted to #50107